src/docs/sphinx/dev_guide/dev_summary.rst

```diff
@@ -72,6 +72,97 @@ See :ref:`pullrequest-label` for a description of our review process and
 how we use pull requests.
 
 
+======================================================
+Contributors and Project Access
+======================================================
+
+Axom maintains three levels of project access on it GitHub project:
+
+  * **Core team members.** Individuals on the core Axom team are frequent
+    Axom contributors and participate regularly in project meetings,
+    discussions, and other project activities. They are members of
+    the LLNL GitHub organization and the ``axom`` GitHub team. Their
+    project privileges include the ability to create branches in the repository,
+    push code changes to the Axom repo, make PRs, and merge them when they are
+    approved and all checks have passed.
+  * **Regular contributors.** Individuals, who are not on the core Axom team,
+    but are members of the LLNL GitHub organization and are involved in some
+    aspects of Axom development are considered regular contributors. They are
+    members of the ``axom-contrib`` GitHub team. Their project privileges
+    include the ability to create branches in the repository, push code changes
+    to the Axom repo, and make PRs. However, they may not merge PRs and must
+    coordinate with the core team to have their work included in the develop
+    branch. This is mainly due to the way GitHub structures its project
+    access levels.
+  * **Everyone else.** Anyone with a GitHub account is welcome to contribute
+    to Axom. Individuals outside of the two groups described above, and 
+    specifically not a member of LLNL GitHub organization, can make PRs
+    in the Axom project, but must do so from a branch on a *fork* of
+    the Axom repo. Thus, the process of reviewing and merging contributions
+    involves additional steps which we describe here.
+
+--------------------------
+Forking the repository
+--------------------------
+
+The requirement for individuals outside of the LLNL GitHub organization
+to contribute on a fork of the repo is due to policies enforced
+by the LLNL organization on GitHub (in which the Axom project resides) and the
+Livermore Computing (LC) organization (in which we run our GitLab CI testing).
+Fortunately, you may still contribute to Axom by `forking the Axom repo
+<https://github.com/LLNL/axom/fork>`_. Forking creates a copy of the Axom
+repository that you own. You can make changes on your local copy and push them
```

Comment:
Missing a "to", "push them _to_ your fork on GitHub"

Reply:
Fixed.